src/main/webapp/js/script.js

```diff
@@ -76,3 +76,58 @@ function showOrHideNotes(version) {
     $('div.setup-panel div a.btn-success').trigger('click');
 });
 
+// Enroll Form Validation using bootstrapValidator Framework
+
+$(document).ready(function(){
+    $(".enrollForm").bootstrapValidator({
+
+        message: "This value is not valid",
+        feedbackIcons: {
+                valid: 'glyphicon glyphicon-ok',
+                invalid: 'glyphicon glyphicon-remove',
+                validating: 'glyphicon glyphicon-refresh'
+            },
+
+        // Validation of fields form
+        fields: {
+            currentVersion: {
+              message: 'The username is not valid',
+                validators: {
+                    notEmpty: {
+                        message: 'The current version is required and cannot be empty'
+                    },
+                    stringLength: {
+                        min: 6,
+                        max: 6,
+                        message: 'The current version must be 6 characters long'
+                    },
+                    //Regular Expression to format the semantic version
+                    regexp: {
+                        regexp: /^(\d+\.)?(\d\.)?(\*|\d)$/,
+                        message: 'The current version can only consist of a symantic version'
```

Comment:
-> semantic. Also, what does this mean? :).
Maybe you could instead say, Incorrect format:  Version should contain a major, minor and patch version numbers separated by a '.'